src/frontend/js/modules/session.js

```diff
@@ -121,6 +121,12 @@
             });
         })
         .catch(error => {
+          $log.error('error while bootstrapping user session', error);
+
+          if (error.code && error.code === 401) {
+            esnAuth.signout();
```

Comment:
should we return here? Because we don't need the following `apply` block if the user is signed out, right?